main_test.go

```diff
@@ -119,6 +122,7 @@ func TestEmptyOptionValues(t *testing.T) {
         OPTION_RELEASE_ASSET_CHECKSUM,
         OPTION_RELEASE_ASSET_CHECKSUM_ALGO,
         OPTION_GITHUB_API_VERSION,
+        OPTION_WITH_PROGRESS,
```

Comment:
Thanks for adding the flag! Could you update the README too?

Reply:
wrote the usage documentation as an example. Good?